docs/api/protocol-buffers.md

```diff
@@ -23,7 +23,7 @@ To use protobuf as a request/input to the HTTP API you need to do two things:
 * Send the proper HTTP header to signal a protobuf payload. The header should be: `Content-Type: application/x-protobuf`
 * Send protocol buffer's serialized bytes as the body of the HTTP request
 
-The message we use for the entire transaction is the `Api` message, whose definition you can find [here](../../proto/api.proto). All of the request parameters should be filled out via the `Options` message attached to the `Api` message. Most importantly, you will want to set your `format` to `pbf` and your `action` to the relevant API you are calling (though the HTTP request path also provides the latter). The rest of the request options depend on which API you are calling. For more information about what and which options to set for a given API please read that APIs specific docs regarding its request options.
+The message we use for the entire transaction is the `Api` message, whose definition you can find [here](../../proto/api.proto). All of the request parameters should be filled out via the `Options` message attached to the `Api` message. Most importantly, you will want to set your `format` to `pbf` and your `action` to the relevant API you are calling (though the HTTP request path also provides the latter). The `options` object also contains a subobject named `pbf_field_selector` which can be used to turn on/off the top level fields in the response. For example, if you only want the `directions` part of the protobuf response to be present (much smaller payload) then turn on only that flag in the field selector. The rest of the request options depend on which API you are calling. For more information about what and which options to set for a given API please read that APIs specific docs regarding its request options.
```

Comment:
just a quick blerb about how to select only the parts of the response you want to have returned to you rather than the whole thing (which is the default)